packages/tests/src/lib/service_node.ts

```diff
@@ -4,13 +4,11 @@ import { Multiaddr, multiaddr } from "@multiformats/multiaddr";
 import { DefaultPubsubTopic } from "@waku/interfaces";
 import { isDefined } from "@waku/utils";
 import { Logger } from "@waku/utils";
-import { bytesToHex, hexToBytes } from "@waku/utils/bytes";
 import pRetry from "p-retry";
 import portfinder from "portfinder";
 
 import {
   Args,
-  KeyPair,
   LogLevel,
   MessageRpcQuery,
```

Comment:
Can see a bunch of "rpc" references here and there. Perhaps rename to rest? eg `MessageRpcQuery`?